sdv/multi_table/hma.py

```diff
@@ -176,26 +195,35 @@ def _pop_foreign_keys(self, table_data, table_name):
 
         return keys
 
-    def _model_tables(self, augmented_data):
+    def _model_tables(self, augmented_data, prefix=None):
         """Model the augmented tables.
 
         Args:
             augmented_data (dict):
                 Dictionary mapping each table name to an augmented ``pandas.DataFrame``.
+            prefix (str):
```

Comment:
Why is this parameter necessary? It seems like this method is only called in one place and it always provides the prefix as `'\n'`. Can't we just set it in here?

Reply:
🤔 yeah we could assumingly we always use `_model_tables` from other methods.